web-portal/components/EventCard.vue

```diff
@@ -141,6 +144,7 @@
           this.calendar_event.tags.includes('condition:sold-out')
       },
       isRemote: function () {
+        console.log(this.calendar_event.title + ' - ' + JSON.stringify(this.calendar_event.tags))
```

Comment:
we should delete this before we check in. It's not great to leave in console logs, they can degrade performance if nothing esle

Reply:
Removed, thanks for flagging.